src/utils/replicate/mixtral_api.ts

```diff
@@ -13,25 +27,48 @@ export interface InputType {
     prompt_template: string;
 }
 
-export const DEFAULT_MIXTRAL_VALUES = {
-    max_new_tokens: 1024,
-    temperature: 0.6,
-    top_p: 0.9,
-    top_k: 50,
-    presence_penalty: 0,
-    frequency_penalty: 0,
-    prompt_template: '<s>[INST] {prompt} [/INST] ',
-};
+
+
+function shrink_message(message: string): [string, string] {
```

Comment:
May be nice un its own utile file in case other commands need it 🤔
+ Return tuple of two strings, mixtral can't answer with a 5k long message ?

Reply:
I move it to utils 👍
about the return : I prefer return a partition than a list of multiple `< max_length` string. Mixtral only send token about 1 to 5 characters so the use case of a list is not there.